neutrino-custom/lint.js

```diff
@@ -1,13 +1,11 @@
-'use strict';
```

Comment:
This `'use strict'` is still needed, since this file (and the others under `neutrino-custom/*` is run by node.js, rather than by the browser - and node doesn't yet support modules natively (other than when started with `--experimental-modules`), and even then the file would need to have file extension `.mjs` to be treated as a module (which is currently a point of contention within the node / JS community). More info here:
https://nodejs.org/api/esm.html

As such in my local Neutrino branch I do something like:
```js
// `use strict` is still necessary here since this file is not treated as a module.
'use strict'; // eslint-disable-line strict, lines-around-directive

// ...
```

Reply:
Ahh, gotcha.  OK.  fixed.